server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -484,10 +485,11 @@ public void sessionExpired() {
         }
 
       } catch (Throwable t) {
+        String msg = "Error recovering tablet " + extent + " from log files";
         if (tableConfiguration.getBoolean(Property.TABLE_FAILURES_IGNORE)) {
-          log.warn("Error recovering from log files: ", t);
+          log.warn(msg, t);
         } else {
-          throw new RuntimeException(t);
+          throw new RuntimeException(msg, t);
```

Comment:
I'm not comfortable with how we're downgrading a potential `Error` type (which is a `Throwable`) to a mere RTE, or even worse... just a log message. I think we should let `Error` fall through, and only catch `Exception`, or better yet, `CheckedException1 | CheckedException2 | RuntimeException`, and perhaps a more specific RTE could be used?

Reply:
I am not sure why we are wrapping it in a runtime... I just added a more informative message.

Reply:
Since we are catching Throwable, I think we should throw a throwable.